include/curl/curl.h

```diff
@@ -1015,6 +1015,15 @@ typedef enum {
 #define CURLALTSVC_H2           (1<<4)
 #define CURLALTSVC_H3           (1<<5)
 
+/* bitmask values for CURLOPT_UPLOAD_FLAGS */
+#define CURLUPLOADFLAG_ANSWERED (1<<0)
+#define CURLUPLOADFLAG_DELETED  (1<<1)
+#define CURLUPLOADFLAG_DRAFT    (1<<2)
+#define CURLUPLOADFLAG_FLAGGED  (1<<3)
+#define CURLUPLOADFLAG_SEEN     (1<<4)
```

Comment:
Can we try using just `UL` instead of `UPLOAD` in these names or something to try to shorten them a little? They are long!